application/jaxrs-activemq-quarkus/src/main/java/webapp/tier/service/ActiveMqConsumer.java

```diff
@@ -0,0 +1,36 @@
+package webapp.tier.service;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.jms.JMSConsumer;
+import javax.jms.JMSException;
+import javax.jms.Message;
+import javax.jms.TextMessage;
+
+import org.eclipse.microprofile.config.ConfigProvider;
+
+import webapp.tier.bean.MsgBean;
+import webapp.tier.service.socket.ActiveMqSocket;
+import webapp.tier.util.MsgUtils;
+
+@ApplicationScoped
+public class ActiveMqConsumer {
+
+	private static final Logger LOG = Logger.getLogger(ActiveMqConsumer.class.getSimpleName());
+	private static String splitkey = ConfigProvider.getConfig().getValue("activemq.split.key", String.class);
+	ActiveMqSocket amqsock = new ActiveMqSocket();
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/yurake/k8s-3tier-webapp/pullRequest?prid=6118916)